src/Messages/OWSMessageSender.m

```diff
@@ -257,7 +257,7 @@ @interface OWSMessageSender ()
 @property (nonatomic, readonly) id<ContactsManagerProtocol> contactsManager;
 @property (nonatomic, readonly) ContactsUpdater *contactsUpdater;
 @property (nonatomic, readonly) OWSDisappearingMessagesJob *disappearingMessagesJob;
-@property (nonatomic, readonly) NSOperationQueue *sendingQueue;
+@property (atomic, readonly) NSMutableDictionary *sendingQueueMap;
```

Comment:
nit: can you type annotate this dictionary.

And why can't it be nonatomic? We're only assigning it in the initializer.